### DISCUSSION TITLE
优先祝福截图帖

### DISCUSSION THREAD

**Discussion by CHNZYX** [CHNZYX/Auto_Simulated_Universe#69] [General]
如果你有希望增加的优先祝福，请在祝福界面截图，保存为jpg并发送到本帖
目前存在的优先事件：火堆外的夜，戒律性闪变

**Comment by Eric-Joker** [comment]
必须算上回归不等式啊（
但是看源码好像已经有相关逻辑了？只是看到了这个祝福名没看逻辑。ocr.py line89。

**Comment by radi-anter** [comment]
能不能加一个优先选择没选过的祝福、奇物的功能？开图鉴用。
